Electron/src/global/global.ts

```
@@ -16,5 +18,16 @@ namespace Global {
     canGoBack: boolean | undefined;
     canGoForward: boolean | undefined;
   }
+
+  // Get version from package.json or use fallback
+  const APP_VERSION = packageJson?.version
+    ? `v${packageJson.version}`
+    : 'v1.0.0';
```

Don't use a valid version as default value because it can be misleading. Use something like x.xx